src/pages/DetailsPage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { IoIosClose } from 'react-icons/io';
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { HiSearchCircle } from 'react-icons/hi';
+import { getAirQuality, getError, getStatus } from '../redux/airQualitySlice';
+import DetailsSummary from '../components/DetailsSummary';
+import Forcast from '../components/Forcast';
+import ApComponents from '../components/ApComponents';
+
+const DetailsPage = () => {
+  const airQuality = useSelector(getAirQuality);
+  const status = useSelector(getStatus);
+  const error = useSelector(getError);
+
+  return (
+    <div className="relative max-w-[440px] w-[100vw] min-h-[100vh] mx-auto bg-gray-100">
+      <div className="flex justify-between items-center px-4 pt-3">
+        <Link to="/">
+          <div className="font-bold text-black text-2xl border border-solid border-gray-300 rounded-full p-1 w-8 h-8 flex flex-col justify-center items-center">
+            <IoIosClose />
+          </div>
+        </Link>
+        <Link to="/search">
+          <HiSearchCircle className="text-4xl text-gray-300 " />
+        </Link>
+      </div>
+      <div className="px-5 py-7">
+        <DetailsSummary airQuality={airQuality} status={status} error={error} summary />
+        <Forcast airQuality={airQuality} />
+        <ApComponents airQuality={airQuality} />
+      </div>
+    </div>
+  );
+};
+
+export default DetailsPage;
```

Comment:
- [ ] Your web app sounds good in your video presentation, but when I tested it locally I encountered issues when I want to navigate to a detail page of a location. It gives me the following error:

<img width="330" alt="image" src="https://user-images.githubusercontent.com/77190203/194704775-15675017-f07e-443d-91d4-cbead9408a69.png">

Reply:
This is a error.message thrown to the UI. 

This tells us that the is a network issue from you end make cannot connect the local server to the API to fulfil a request, as thus the request was rejected and an error was throw back to the UI. 

![Code Snap 2](https://user-images.githubusercontent.com/95294594/194706842-a964c730-8c08-459d-82d7-7d50baed588f.png)

Reply:
Thanks again, @medaminedev66 . I realized that the API links wasn't a secured link and thus, it was blocked. HTTP:// was replaced with HTTPS://

          const APP_ID = '05cac1b6e91db3212d68b7e40d3755c6';
          const AP_BASE_URL = 'https://api.openweathermap.org/data/2.5/air_pollution?';
          const FORCAST_AP_BASE_URL =
            'https://api.openweathermap.org/data/2.5/air_pollution/forecast?';
          const GEO_BASE_URL = 'https://api.openweathermap.org/geo/1.0/direct?';
          const limit = 5;

Again, thanks for the good eye.

Reply:
Hi @uisendy, great work fixing this error 👏. Now the detail page is displayed without errors and I can see the air quality of my city 😄.